diff --git a/platform/fpga/cva6-altera/platform.c b/platform/fpga/cva6-altera/platform.c
index a0d13f2..573c5cd 100644
--- a/platform/fpga/cva6-altera/platform.c
+++ b/platform/fpga/cva6-altera/platform.c
@@ -2,7 +2,7 @@
 /*
  * Copyright (C) 2025 PlanV Technologies
  *				Angela Gonzalez
- * Adapted from Ariane platform
+ * Adapted from CVA6 platform
  */
 
 #include <sbi/riscv_asm.h>
@@ -19,59 +19,59 @@
 #include <sbi_utils/serial/uart8250.h>
 #include <sbi_utils/timer/aclint_mtimer.h>
 
-#define ARIANE_UART_ADDR			0x10000000
-#define ARIANE_UART_FREQ			50000000
-#define ARIANE_UART_BAUDRATE			115200
-#define ARIANE_UART_REG_SHIFT			0
-#define ARIANE_UART_REG_WIDTH			1
-#define ARIANE_PLIC_ADDR			0xc000000
-#define ARIANE_PLIC_NUM_SOURCES			3
-#define ARIANE_HART_COUNT			1
-#define ARIANE_CLINT_ADDR			0x2000000
-#define ARIANE_ACLINT_MTIMER_FREQ		1000000
-#define ARIANE_ACLINT_MSWI_ADDR			(ARIANE_CLINT_ADDR + \
+#define CVA6_UART_ADDR			0x10000000
+#define CVA6_UART_FREQ			50000000
+#define CVA6_UART_BAUDRATE			115200
+#define CVA6_UART_REG_SHIFT			0
+#define CVA6_UART_REG_WIDTH			1
+#define CVA6_PLIC_ADDR			0xc000000
+#define CVA6_PLIC_NUM_SOURCES			3
+#define CVA6_HART_COUNT			1
+#define CVA6_CLINT_ADDR			0x2000000
+#define CVA6_ACLINT_MTIMER_FREQ		1000000
+#define CVA6_ACLINT_MSWI_ADDR			(CVA6_CLINT_ADDR + \
 						 CLINT_MSWI_OFFSET)
-#define ARIANE_ACLINT_MTIMER_ADDR		(ARIANE_CLINT_ADDR + \
+#define CVA6_ACLINT_MTIMER_ADDR		(CVA6_CLINT_ADDR + \
 						 CLINT_MTIMER_OFFSET)
 
 static struct plic_data plic = {
-	.addr = ARIANE_PLIC_ADDR,
-	.num_src = ARIANE_PLIC_NUM_SOURCES,
+	.addr = CVA6_PLIC_ADDR,
+	.num_src = CVA6_PLIC_NUM_SOURCES,
 };
 
 static struct aclint_mswi_data mswi = {
-	.addr = ARIANE_ACLINT_MSWI_ADDR,
+	.addr = CVA6_ACLINT_MSWI_ADDR,
 	.size = ACLINT_MSWI_SIZE,
 	.first_hartid = 0,
-	.hart_count = ARIANE_HART_COUNT,
+	.hart_count = CVA6_HART_COUNT,
 };
 
 static struct aclint_mtimer_data mtimer = {
-	.mtime_freq = ARIANE_ACLINT_MTIMER_FREQ,
-	.mtime_addr = ARIANE_ACLINT_MTIMER_ADDR +
+	.mtime_freq = CVA6_ACLINT_MTIMER_FREQ,
+	.mtime_addr = CVA6_ACLINT_MTIMER_ADDR +
 		      ACLINT_DEFAULT_MTIME_OFFSET,
 	.mtime_size = ACLINT_DEFAULT_MTIME_SIZE,
-	.mtimecmp_addr = ARIANE_ACLINT_MTIMER_ADDR +
+	.mtimecmp_addr = CVA6_ACLINT_MTIMER_ADDR +
 			 ACLINT_DEFAULT_MTIMECMP_OFFSET,
 	.mtimecmp_size = ACLINT_DEFAULT_MTIMECMP_SIZE,
 	.first_hartid = 0,
-	.hart_count = ARIANE_HART_COUNT,
+	.hart_count = CVA6_HART_COUNT,
 	.has_64bit_mmio = TRUE,
 };
 
 /*
- * Ariane platform early initialization.
+ * CVA6 platform early initialization.
  */
-static int ariane_early_init(bool cold_boot)
+static int CVA6_early_init(bool cold_boot)
 {
 	/* For now nothing to do. */
 	return 0;
 }
 
 /*
- * Ariane platform final initialization.
+ * CVA6 platform final initialization.
  */
-static int ariane_final_init(bool cold_boot)
+static int CVA6_final_init(bool cold_boot)
 {
 	void *fdt;
 
@@ -85,20 +85,20 @@ static int ariane_final_init(bool cold_boot)
 }
 
 /*
- * Initialize the ariane console.
+ * Initialize the CVA6 console.
  */
-static int ariane_console_init(void)
+static int CVA6_console_init(void)
 {
-	return altera_uart_init(ARIANE_UART_ADDR,
-			     ARIANE_UART_FREQ,
-			     ARIANE_UART_BAUDRATE,
-			     ARIANE_UART_REG_SHIFT,
-			     ARIANE_UART_REG_WIDTH);
+	return altera_uart_init(CVA6_UART_ADDR,
+			     CVA6_UART_FREQ,
+			     CVA6_UART_BAUDRATE,
+			     CVA6_UART_REG_SHIFT,
+			     CVA6_UART_REG_WIDTH);
 }
 
-static int plic_ariane_warm_irqchip_init(int m_cntx_id, int s_cntx_id)
+static int plic_CVA6_warm_irqchip_init(int m_cntx_id, int s_cntx_id)
 {
-	size_t i, ie_words = ARIANE_PLIC_NUM_SOURCES / 32 + 1;
+	size_t i, ie_words = CVA6_PLIC_NUM_SOURCES / 32 + 1;
 
 	/* By default, enable all IRQs for M-mode of target HART */
 	if (m_cntx_id > -1) {
@@ -121,9 +121,9 @@ static int plic_ariane_warm_irqchip_init(int m_cntx_id, int s_cntx_id)
 }
 
 /*
- * Initialize the ariane interrupt controller for current HART.
+ * Initialize the CVA6 interrupt controller for current HART.
  */
-static int ariane_irqchip_init(bool cold_boot)
+static int CVA6_irqchip_init(bool cold_boot)
 {
 	u32 hartid = current_hartid();
 	int ret;
@@ -133,13 +133,13 @@ static int ariane_irqchip_init(bool cold_boot)
 		if (ret)
 			return ret;
 	}
-	return plic_ariane_warm_irqchip_init(2 * hartid, 2 * hartid + 1);
+	return plic_CVA6_warm_irqchip_init(2 * hartid, 2 * hartid + 1);
 }
 
 /*
  * Initialize IPI for current HART.
  */
-static int ariane_ipi_init(bool cold_boot)
+static int CVA6_ipi_init(bool cold_boot)
 {
 	int ret;
 
@@ -153,9 +153,9 @@ static int ariane_ipi_init(bool cold_boot)
 }
 
 /*
- * Initialize ariane timer for current HART.
+ * Initialize CVA6 timer for current HART.
  */
-static int ariane_timer_init(bool cold_boot)
+static int CVA6_timer_init(bool cold_boot)
 {
 	int ret;
 
@@ -172,20 +172,20 @@ static int ariane_timer_init(bool cold_boot)
  * Platform descriptor.
  */
 const struct sbi_platform_operations platform_ops = {
-	.early_init = ariane_early_init,
-	.final_init = ariane_final_init,
-	.console_init = ariane_console_init,
-	.irqchip_init = ariane_irqchip_init,
-	.ipi_init = ariane_ipi_init,
-	.timer_init = ariane_timer_init,
+	.early_init = CVA6_early_init,
+	.final_init = CVA6_final_init,
+	.console_init = CVA6_console_init,
+	.irqchip_init = CVA6_irqchip_init,
+	.ipi_init = CVA6_ipi_init,
+	.timer_init = CVA6_timer_init,
 };
 
 const struct sbi_platform platform = {
 	.opensbi_version = OPENSBI_VERSION,
 	.platform_version = SBI_PLATFORM_VERSION(0x0, 0x01),
-	.name = "ARIANE RISC-V",
+	.name = "CVA6 RISC-V",
 	.features = SBI_PLATFORM_DEFAULT_FEATURES,
-	.hart_count = ARIANE_HART_COUNT,
+	.hart_count = CVA6_HART_COUNT,
 	.hart_stack_size = SBI_PLATFORM_DEFAULT_HART_STACK_SIZE,
 	.platform_ops_addr = (unsigned long)&platform_ops
 };
